### PR TITLE
Make Kokoro logs less noisy

### PR DESCRIPTION
**Pull request by matt-kwong**
Using this PR to test changes for now

### CODE REVIEW

File: buildscripts/kokoro/windows.bat [hunk 1]
```diff
@@ -15,14 +17,14 @@ mkdir grpc-java-helper
...
-unzip 7za920.zip
+wget -q http://www.7-zip.org/a/7za920.zip >NUL
+unzip 7za920.zip >NUL
 mv 7za.exe grpc-java-helper
 
 cd grpc-java-helper
 
-call "%VS120COMNTOOLS%\vsvars32.bat"
-call "%WORKSPACE%\buildscripts\make_dependencies.bat"
+call "%VS120COMNTOOLS%\vsvars32.bat" >NUL
```
Comment (ejona86) [inline buildscripts/kokoro/windows.bat]: This is a bit too strong. Most of these things at the top, I was okay with.

File: buildscripts/kokoro/windows.bat [hunk 2]
```diff
@@ -31,12 +33,16 @@ echo failOnWarnings=true>> gradle.properties
 echo vcProtobufLibs=%ESCWORKSPACE%\\grpc-java-helper\\protobuf-%PROTOBUF_VER%\\cmake\\build\\Release>> gradle.properties
 echo vcProtobufInclude=%ESCWORKSPACE%\\grpc-java-helper\\protobuf-%PROTOBUF_VER%\\cmake\\build\\include>> gradle.properties
 
+@echo on
```
Comment (ejona86) [inline buildscripts/kokoro/windows.bat]: Instead of echo on and echo off, just prefix the command with `@`. Like `@cmd.exe ...`

**Comment by matt-kwong** [comment]
Sorry for the delay, I got caught up in other things. I'm not sure how to get the for loop completely silent without using `@echo off`. Otherwise, I think the most of the noise is generated by `call "%WORKSPACE%\buildscripts\make_dependencies.bat"`, but that is at the top and out of the way. Let me know how it looks to you.

File: buildscripts/kokoro/windows.bat [hunk 3]
```diff
@@ -15,9 +15,9 @@ mkdir grpc-java-helper
 @rem Install 7za
 @rem TODO(mattkwong): After Windows tests are no longer running on Jenkins, 7za
 @rem doesn't need to be installed and make_dependencies.bat can use unzip instead
-wget http://www.7-zip.org/a/7za920.zip
-unzip 7za920.zip
-mv 7za.exe grpc-java-helper
+@wget -q http://www.7-zip.org/a/7za920.zip >NUL
```
Comment (ejona86) [inline buildscripts/kokoro/windows.bat]: Let's leave these echo'd. My main concern is loops and things that spew a lot. These are just a few lines. The `>NUL` for unzip is probably appropriate though.

**Pull request merged**